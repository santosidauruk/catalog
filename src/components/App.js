import React, { Component } from 'react'
import { Switch, Route } from 'react-router-dom'

import { ProductList } from './screen'
import '../styles/main.css'

class App extends Component {
    render() {
        return (
            <Switch>
                <Route exact path="/" component={ProductList} />
                {/* <Route path="/:productName" component={DetailProduct} /> */}
            </Switch>
        )
    }
}

export default App
