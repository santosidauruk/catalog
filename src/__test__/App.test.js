import React from 'react'
import renderer from 'react-test-renderer'
import BrowserRouter from 'react-router-dom/BrowserRouter'
import App from '../components/App'

describe('App testing', () => {
    it('should rendered correctly', () => {
        const component = renderer.create(
            <BrowserRouter>
                <App />
            </BrowserRouter>
        )
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
