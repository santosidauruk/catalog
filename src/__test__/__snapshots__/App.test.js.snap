// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App testing should rendered correctly 1`] = `
<div
  className="sc-bxivhb kYkLkP"
/>
`;
